media_driver/linux/common/ddi/media_libva.h

```diff
@@ -61,7 +61,7 @@
 #if VA_MAJOR_VERSION < 1
 #define DDI_CODEC_GEN_MAX_DISPLAY_ATTRIBUTES       4
 #else
-#define DDI_CODEC_GEN_MAX_DISPLAY_ATTRIBUTES       0    // set it to zero, unsupported.
+#define DDI_CODEC_GEN_MAX_DISPLAY_ATTRIBUTES       1
```

Comment:
How about change to DDI_MEDIA_GEN_MAX_DISPLAY_ATTRIBUTES

Reply:
accept